JS/Script.js

```diff
@@ -1,73 +1,93 @@
 const bookCollection = document.querySelector('#collection');
 const addBook = document.querySelector('#add-book');
-
+let library;
 if (localStorage.getItem('Awesome Book Collection') === null) {
-  var library = [];
+  library = [];
 } else {
-  var library = JSON.parse(localStorage.getItem('Awesome Book Collection'))
+  library = JSON.parse(localStorage.getItem('Awesome Book Collection'));
 }
 
 function storeEntryData(titleElement, authorElement) {
-  let entryObject = {};
+  const entryObject = {};
   entryObject.title = titleElement.value;
   entryObject.author = authorElement.value;
   library.push(entryObject);
 }
 
-function appendBookHTML(title = false, author = false) {  
-  if (title === false || author === false) { return }
-  let newBook = document.createElement('div');
-  let newTitle = document.createElement('div');
-  newTitle.innerHTML = title;
+function appendBookHTML(title = false, author = false) {
+  if (title === false || author === false) {
+    return true;
+  }
+  const newBook = document.createElement('div');
+  newBook.setAttribute('class', 'book-list');
+  const newTitle = document.createElement('div');
+  newTitle.innerHTML = title.concat('-by-');
   newBook.appendChild(newTitle);
-  let newAouthor = document.createElement('div');
+  const newAouthor = document.createElement('div');
   newAouthor.innerHTML = author;
   newBook.appendChild(newAouthor);
-  let removeButton = document.createElement('div');
+  const removeButton = document.createElement('div');
   removeButton.innerHTML = '<button class="remove">Remove</button><br><br>';
   newBook.appendChild(removeButton);
   bookCollection.appendChild(newBook);
-} 
+  return false;
+}
 
 function loadBooks() {
-  for (let i = 0; i < library.length; i++) {    
-    appendBookHTML(library[i].title, library[i].author)
+  for (let i = 0; i < library.length; i += 1) {
+    appendBookHTML(library[i].title, library[i].author);
   }
 }
 
 loadBooks();
 
 function updateNewBook() {
-  appendBookHTML(library[library.length-1].title, library[library.length-1].author);
+  appendBookHTML(library[library.length - 1].title, library[library.length - 1].author);
   localStorage.setItem('Awesome Book Collection', JSON.stringify(library));
 }
 
 function removeBook(element) {
```

Comment:
Please add `methods` to `add` and `remove` books in a class and make sure the app functionality works as expected after refactring the code.